src/java/com/puppetlabs/http/client/SyncHttpClient.java

```diff
@@ -12,6 +15,16 @@
 public interface SyncHttpClient extends Closeable {
 
     /**
+     * @return a SortedMap of metric name to Timer instance
+     */
+    public SortedMap<String, Timer> getClientMetrics();
+
+    /**
+     * @return a Map of metric name to metric data
+     */
+    public Map<String, Map<String, Object>> getClientMetricsData();
```

Comment:
`SortedMap` for the first because that was the return type of the underlying method from the dropwizard library. I can make both of these `Map` for consistency though.